Jetcaster/app/build.gradle.kts

```diff
@@ -109,6 +110,7 @@ dependencies {
 
     implementation(libs.androidx.window)
 
+    implementation(libs.accompanist.adaptive)
```

Comment:
@alexvanyo So two pane will be part of accompanist adaptive?

Reply:
Yes, at least for the time being to explore its API shape and get feedback.